src/lib.rs

```diff
@@ -80,6 +88,44 @@ impl Write for Vga {
     }
 }
 
+pub static BUFFER: Mutex<Vga> = Mutex::new(
+    Vga {
+        location: 0xb8000 as *mut u8,
+        buffer: [0; ROWS * COLS * 2],
+        position: 0,
+    }
+);
+
+/// Prints something to the screen, with a trailing newline.
+///
+/// # Examples
+///
+/// ```
+/// kprintln!("Hello, world!");
+/// ```
+#[macro_export]
+macro_rules! kprintln {
```

Comment:
OR: spend 5min trying to figure out how to get rid of it, and then if we can at least put it all the way at the end